make test has bug

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
 make test
**What you expected to happen**:
# github.com/carina-io/carina/pkg/devicemanager [github.com/carina-io/carina/pkg/devicemanager.test]
pkg/devicemanager/manager_test.go:94:24: not enough arguments in call to NewDeviceManager
**How to reproduce it**:
dm := NewDeviceManager("localhost", nil, stopChan)
**Anything else we need to know?**:

**Environment**:
- CSI Driver version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/.gitignore
+++ b/.gitignore
@@ -24,3 +24,4 @@ bin
 *~
 
 vendor/
+.vscode/

--- a/pkg/devicemanager/manager_test.go
+++ b/pkg/devicemanager/manager_test.go
@@ -91,7 +91,7 @@ func TestDeviceManager(t *testing.T) {
 
 	stopChan := make(chan struct{})
 	defer close(stopChan)
-	dm := NewDeviceManager("localhost", stopChan)
+	dm := NewDeviceManager("localhost",nil, stopChan)
 	defer func() {
 		// 清理volume
 		_ = cleanVolume(dm)

